src/components/UserManagement/UserTableData.jsx

```diff
@@ -142,6 +153,7 @@ const UserTableData = React.memo(props => {
       <td>
         <button
           type="button"
+          disabled={isOwner? false : true}
```

Comment:
```suggestion
```
Also unrelated to the "create new users" permission.

Reply:
> Also unrelated to the "create new users" permission.

I have updated the code. But we need to disable these.

Please check the feature description:
If the owner gives users like (Owners, Administrators, Core Team, Manager, Mentors, Volunteers) “create new users” access in “permission management, then this individual should see “user management” button, and be able to create new users, while at the same time, cannot use “delete”, “reset password”, “set final day”, “pause”, “request time off” these modification functionalities.

If the owner revokes anyone’s “create new users” access in “permission management”, then the person cannot see “user management” button.

Reply:
"create new users" is one of the several permissions which grant the ability to see "user management". Other permissions pertaining to those separate actions (“delete”, “reset password”, “set final day”, “pause”, “request time off”) should grant access to the page and the ability to perform those actions, but should stay separate from the "create new users" permission.